src/main/java/dev/dfonline/codeclient/dev/BuildPhaser.java

```diff
@@ -151,6 +144,17 @@ private void finishClipping() {
             abilities.flying = wasFlying;
             waitForTP = true;
 
+            var size = plot.assumeSize();
+
+            var x = Math.min(Math.max(player.getX(), plot.getX()), plot.getX() + size.size + 1);
+            var y = player.getY();
+            var z = Math.min(Math.max(player.getZ(), plot.getZ()), plot.getZ() + size.size + 1);
+            var pitch = player.getPitch();
+            var yaw = player.getYaw();
+
+            player.networkHandler.sendChatCommand(String.format("ptp %s %s %s %s %s", x, y, z, pitch, yaw));
```

Comment:
Remind me to send you the queue and ratelimiting thing for commands to keep codeclient as an intrusive as it has been